esileapclient/osc/v1/lease_offer.py

```diff
@@ -35,6 +35,13 @@ def get_parser(self, prog_name):
             required=False,
             help="Time when the offer will expire and no longer be "
                  "'available'.")
+        parser.add_argument(
+            '--name',
+            dest='name',
+            required=False,
+            help="Name of the contract being created. "
+                 "Can be used in place of uuid for contract management "
```

Comment:
Ditto here; also, you probably mean 'offer' instead of 'contract'

Reply:
fixed